config/crd/bases/actions.summerwind.dev_runners.yaml

```diff
@@ -5196,6 +5208,14 @@ spec:
                   type: string
                 message:
                   type: string
+                owner:
+                  type: string
+                repository:
+                  type: string
+                repo:
+                  type: string
+                workflow:
```

Comment:
Nit but I would prefer `workflow_name` so that it corresponds with the `name` field of the workflow defintiion